mpas_analysis/shared/plot/plotting.py

```diff
@@ -761,7 +761,10 @@ def plot_vertical_section(
     ylabel=None,
     fileout='moc.png',
     figsize=(10, 4),
-    dpi=300):  # {{{
+    dpi=300,
+    xLim=None,
+    yLim=None,
+    invertYAxis= True):  # {{{
```

Comment:
@milenaveneziani, I think the reason @mark-petersen made `True` the default is because that was the behavior before he added the option, see the changes below.  So shouldn't break MOC.